public/apps/configuration/panels/role-view/index-permission-panel.tsx

```diff
@@ -97,7 +100,14 @@ function getColumns(
     },
     {
       field: 'dls',
-      name: 'Document-level security',
+      name: (
+        <EuiToolTip content={ToolTipContent.DocumentLevelSecurity}>
+          <span>
+            Document-level security{' '}
+            <EuiIcon size="s" color="subdued" type="questionInCircle" className="eui-alignTop" />
+          </span>
+        </EuiToolTip>
+      ),
```

Comment:
As role listing page also need this rendering, please have a util function for it.

Reply:
created util function.